src/components/TimeIndicator/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Time indicator.
+ * @copyright IBM Security 2019
+ */
+
+import Time16 from '@carbon/icons-react/lib/time/16';
+
+import classnames from 'classnames';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('time-indicator');
+
+/**
+ * Time indicators display an estimated amount of time it takes to complete a flow. If there is unavailable space to accommodate the text, `minute` should be abbreviated to `min`.
+ */
```

Comment:
I like this info but I wonder if it shouldn't have more visibility in the storybook demo instead? (I don't think folks expect to find this type of usage info in source code)

Reply:
Absolutely! See https://github.com/carbon-design-system/ibm-security/pull/240/files#diff-c736752891da25bcea8e973d2b3a5798R21

When spiking the `addon-docs` work, I discovered that the plugin will generate the documentation based on the JSDoc for the component itself. The dream is to have the usage information contained within the component definition and pulled from there but unfortunately, there are still outstanding issues with integrating `addon-docs` with our current setup (see https://github.com/storybookjs/storybook/issues/8128).

This is more future-proofing it so it's ready for that integration, but I'm happy to remove this for now if that's thinking too far ahead.